readme.md

```diff
@@ -215,16 +215,18 @@ Type1|Type2               // Union
 
 > [See Docs](https://developers.google.com/closure/templates/docs/commands#switch "Switch")
 
-###### elvis
+###### ternary
 
 ```
-Hello {$name ?: 'there'}
+{isNonnull($name) ? 'Hello {$name}' : 'Hello there'}
```

Comment:
```
{$a ?: 'c'}
{$a ? 'b' : 'c'}
{isNonnull($a) ? 'b' : 'c'}
```

becomes in JS (without dynamic function support introduced by Metal):

```js
// {$a ?: 'c'}
var dyn0 = ($$temp = opt_data.a) == null ? 'c' : $$temp;
itext(dyn0);
// {$a ? 'b' : 'c'}
var dyn1 = opt_data.a ? 'b' : 'c';
itext(dyn1);
// {isNonnull($a) ? 'b' : 'c'}
var dyn2 = opt_data.a != null ? 'b' : 'c';
itext(dyn2);
```

As you can see `?:` uses the `isNonnull` check.